frontend/pages/views/MapPageView.js

```diff
@@ -0,0 +1,285 @@
+
+
+            for (const mapSquare of mapData) {
+                // This code right here might cause problems if said user hasn't run syncdb
+                const roughCoords = mapSquare.coordinates;
+
+                // If the map square has coordinates in the spreadsheet,
+                // it pulls those coordinates and makes those the coordinates of the marker
+                // Coords must be in (lat, lng)
+
+                // If the map square does not have the coordinates it sets them to (0, 0)
+                // NOTE(ra): this no longer happens, so we can probably remove this safety check
+                if (roughCoords) {
+                    const roughCoordsList = roughCoords.split(", ");
+                    const lat = parseFloat(roughCoordsList[0]);
+                    const lng = parseFloat(roughCoordsList[1]);
+                    mapSquare.topLeftCoords = {
+                        lat,
+                        lng
+                    };
+                } else {
+                    const lat = 0.0;
+                    const lng = 0.0;
+                    mapSquare.topLeftCoords = {
+                        lat,
+                        lng
+                    };
+                }
+            }
+
+            this.setState({
+                mapData,
+                loading: false
+            });
+
+
+        } catch (e) {
+            console.log(e);
+        }
+        try {
+            const geojsonResponse = await fetch("/api/arrondissements_geojson/");
+            const geojsonData = await geojsonResponse.json();
+
+            //Fetches the data set containing which map squares belong to which arrondissement
+            const mapSquareDataResponse = await fetch("/api/arrondissements_map_squares/");
+            const mapSquareData = await mapSquareDataResponse.json();
+            //Makes a set of all map squares that actually contain pictures. Eventually shouldn't
+            //be needed once full picture database is added.
+            const filledMapSquares = new Set();
+
+            for(const mapSquare of this.state.mapData){
+                if(mapSquare["num_photos"] > 0) {
+                    filledMapSquares.add(mapSquare["id"]);
+                }
+            }
+
+            this.setState({
+                filledMapSquares: filledMapSquares,
+                arrondissementData: mapSquareData["arrondissements"],
+                geojsonData: geojsonData["features"],
+                loading: false
+            });
+        } catch (e) {
+            console.log(e);
+        }
+    }
+
+    render() {
+        if (!this.state.mapData || !this.state.arrondissementData || !this.state.filledMapSquares) {
+            return (<LoadingPage/>);
+        }
+
+        return (<div>
+                <Container fluid className="page-style">
+                    <Row className="page-body">
+                        <Col md={12} lg={7} className="m-0 p-0 min-vh-100" id="map-box">
```

Comment:
This map is currently eating the rest of the page in smaller screen sizes (`md={12}`). I know we want a full-width map but we also want to be able to navigate to the content below it. This might mean two things: first, showing that there IS content below by only allowing the map to take up 70% of the height of the screen instead of all of it. Also setting the `scrollWheelZoom` attribute to false on ParisMap would help (see editable example here: https://react-leaflet.js.org/).